server/runtime/src/main/java/org/infinispan/server/Server.java

```diff
@@ -536,4 +545,24 @@ public ComponentStatus getStatus() {
    public TaskManager getTaskManager() {
       return taskManager;
    }
+
+   @Override
+   public CompletionStage<Path> getServerReport() {
+      return CompletableFuture.supplyAsync(() -> {
+         Path home = serverHome.toPath();
+         ProcessBuilder builder = new ProcessBuilder("sh", "-c", home.resolve("bin/report.sh").toString(), home.toString());
```

Comment:
Is Windows going to be handled? It probably needs to return ```501``` from the REST endpoint

Reply:
I'll do that for non-Linux.